appformer-kogito-bridge/src/main/java/org/appformer/kogito/bridge/client/resource/impl/MockResourceContentService.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.kogito.bridge.client.resource.impl;
+
+import elemental2.promise.Promise;
+import org.appformer.kogito.bridge.client.resource.ResourceContentService;
+
+/**
+ * This {@link ResourceContentService} implementation is used when the envelope API is not available
+ */
+public class MockResourceContentService implements ResourceContentService {
```

Comment:
Agree on having some "no-op" content service, but from my experience calling it as "Mock...Service" does probably not reflect it. Anyway, just a minor comment, just my opinion...

Reply:
Thank you, will rename the class